### ISSUE TITLE
Data entry - IATI organisation dropdown drawing from the wrong org table

### ISSUE THREAD

**Issue by matmaxgeds** [somaliaims/Somali-AIMSUI#477]
The IATI orgs dropdown is drawing from the AIMS orgs table, it needs to draw from the IATI orgs table - this seems to keep breaking, is there some confusion? The AIMS orgs dropdown needs to draw from the AIMS orgs table, and the IATI orgs dropdown needs to draw from the IATI orgs table.

**Comment by raashidahmad** [comment]
Fixed. @matmaxgeds i can see in the back-end data that there are some organizations common in both, but it has been fixed that both the organizations are now coming from their respective table.

**Comment by matmaxgeds** [comment]
@raashidahmad - thanks, and yes there should be some crossover. Now there is a new issue, if I select the IATI org 'aadamiga USA' no IATI activities are found - if we have them as an IATI organisation, there should also be a project for them?

**Comment by raashidahmad** [comment]
@matmaxgeds I discovered that we are not picking up many activities that are no more active. For example, any activity having activity-status as cancelled or closed.
We are extracting and saving organizations in a separate background thread that runs every midnight, extracts and stores IATI Organizations, so they are independently coming regardless of activities when they are queried and parsed using very first screen for data entry. Closing it as data is filtered correctly.
